tests/test_render.py

```diff
@@ -475,6 +475,32 @@ def test_initial_state(self):
 
         assert_clean_console(self)
 
+    def test_array_of_nully_child(self):
+        app = Dash(__name__)
+        app.layout = html.Div(id='nully-wrapper', children=[0])
+
+        self.startServer(app)
+
+        self.wait_for_element_by_css_selector('#nully-wrapper')
+        wrapper = self.driver.find_element_by_id('nully-wrapper')
+
+        self.assertEqual(wrapper.get_attribute('innerHTML'), '0')
+
+        assert_clean_console(self)
+
+    def test_of_nully_child(self):
+        app = Dash(__name__)
+        app.layout = html.Div(id='nully-wrapper', children=0)
```

Comment:
This is the regression case

Reply:
too late for the comment, but this should be merged with the `test_initial_state`.